docker-compose.base.yml

```diff
@@ -1,37 +0,0 @@
-version: '2'
```

Comment:
@cbeer and I chatted and he said this file was just cruft and that we should get rid of it.